site/content/docs/5.0/customize/color.md

```diff
@@ -106,3 +106,40 @@ Here's how you can use these in your Sass:
 ```
 
 [Color utility classes]({{< docsref "/utilities/colors" >}}) are also available for setting `color` and `background-color` using the `500` color values.
+
+## Generating utilities
+
+Bootstrap doesn't include `color` and `background-color` utilities for every individual color, but you can generate these yourself with our [utility API]({{< docsref "/utilities/api" >}}).
+
+1. To start, make sure you've imported our functions, variables, mixins, and utilities.
+2. Then include the `utilities/_color-extended.scss` stylesheet to access a full suite of color Sass maps.
```

Comment:
Stylesheet does not seem right? Also a full suite or the full suite?

Reply:
i'd circumvent the first question by just using "Then include `utilities/_color-extend.scss` to access..."

and in this case, either `a` or `the` work, but i think i'd lean more towards `the`